control-plane/pkg/reconciler/channel/channel.go

```diff
@@ -203,8 +225,20 @@ func (r *Reconciler) FinalizeKind(ctx context.Context, channel *messagingv1beta1
 	return nil
 }
 
+func (r *Reconciler) channelConfigMap() (*corev1.ConfigMap, error) {
```

Comment:
we might change that name later ... once I started w/ the refactoring on all sorts of `reconcileXzy` (like distributed channel)

Reply:
sure, sounds good